src/openforms/registrations/contrib/json_dump/tests/test_backend.py

```diff
@@ -0,0 +1,238 @@
+from base64 import b64decode
+from pathlib import Path
+
+from django.core.exceptions import SuspiciousOperation
+from django.test import TestCase
+
+from glom import glom
+from requests import RequestException
+from zgw_consumers.test.factories import ServiceFactory
+
+from openforms.submissions.public_references import set_submission_reference
```

Comment:
this is not public API and should not be imported outside the submission app :-)